.circleci/config.yml

```diff
@@ -77,6 +83,183 @@ executors:
+          echo "export CLUSTER_1=<< parameters.cluster_1_name >>-${CIRCLE_SHA1:0:7}" >> $BASH_ENV
+          echo "export CLUSTER_2=<< parameters.cluster_2_name >>-${CIRCLE_SHA1:0:7}" >> $BASH_ENV
+
+          # we have to install `unzip` to be able to install aws-cli
+          sudo apt update
+          sudo apt install -y \
+            unzip
+
+    - aws-cli/install
+
+    - aws-eks/install-eksctl
+
+    - aws-eks/install-aws-iam-authenticator
+
+    - kubernetes/install
+
+    - setup_remote_docker:
+        version: 19.03.12
+
+    # Mount files from the upstream jobs
+    - attach_workspace:
+        at: build
+
+    - run:
+        name: "Load Docker images from TAR archives"
+        command: make docker/load
+
+    - aws-ecr/ecr-login:
+        account-url: ECR_REGISTRY
+        region: AWS_DEFAULT_REGION
+
+    - run:
+        name: "Push images to ECR & Create clusters"
+        command: |
+          for image in kuma-cp kuma-dp kuma-init kumactl kuma-universal; do
+            make docker/tag/${image} \
+              BINTRAY_REGISTRY="${ECR_REGISTRY}" \
+              KUMA_VERSION="${TAG}" && \
+            docker push "${ECR_REGISTRY}"/${image}:"${TAG}" &
+          done
+
+          eksctl create cluster \
+            --name "${CLUSTER_1}" \
+            --nodes 1 \
+            &
+          eksctl create cluster \
+            --name "${CLUSTER_2}" \
+            --nodes 1 \
+            &
+
+          FAIL="0"
+          for job in $(jobs -p); do
+            wait "${job}" || FAIL=$((FAIL + 1))
+          done
+
+          echo "${FAIL}"
+
+    - aws-eks/update-kubeconfig-with-authenticator:
+        cluster-name: "${CLUSTER_1}"
+        kubeconfig-file-path: ~/.kube/kind-kuma-1-config
+
+    - aws-eks/update-kubeconfig-with-authenticator:
+        cluster-name: "${CLUSTER_2}"
+        kubeconfig-file-path: ~/.kube/kind-kuma-2-config
+
+    - restore_cache:
+        keys:
+        # prefer the exact match
+        - vm-executor-go.mod-{{ .Branch }}-{{ checksum "go.sum" }}
+
+    - run:
+        name: "Download Go modules"
+        command: go mod download
+
+    # since execution of go commands might change contents of "go.sum",
+    #  we have to save cache immediately
+    - save_cache:
+        key: vm-executor-go.mod-{{ .Branch }}-{{ checksum "go.sum" }}
+        paths:
+        - "/home/circleci/.go-kuma-go"
+
+    - run:
+        name: "Install all development tools"
+        command: make dev/tools
+
+    - run:
+        name: "Setup Helm"
+        command: helm repo add kuma https://kumahq.github.io/charts
+
+    # We need kumactl to be built to run tests, and to not build it again,
+    #  we can use the one already build by earlier jobs
+    - attach_workspace:
+        at: build
+
+    - run:
+        name: "Run E2E tests"
+        command: |
+          export KUMA_UNIVERSAL_IMAGE="${ECR_REGISTRY}/kuma-universal:${TAG}";
+          export E2E_PKG_LIST="./test/e2e/helm";
```

Comment:
so for ECS we are running just HELM suite, not full E2E suite?

Reply:
Yes, that was requirement I received. To have full suite there would have to be much more changes as we are relying for the "universal" part when doing "hybrid" tests on local machine and it expects some local functionalities which I forgot now